src/formatters/stringFormatter.js

```diff
@@ -47,3 +40,139 @@ function invalidOptionsFormatter(results) {
     return output + "\n"
   }, "\n")
 }
+
+function getSeverity(type) {
```

Comment:
I believe this function is a relic of older days and is now unnecessary. Could you try removing it?